src/main/java/core/basesyntax/service/impl/WriterServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.WriterService;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriterServiceImpl implements WriterService {
+    public static final String FILE_NAME = "src/main/resources/report.txt";
+
+    @Override
+    public void createReportFile(String fruitBalanceReport) {
+        File file = new File(FILE_NAME);

```

Comment:
```suggestion
        File file = new File(filePath);

```

It meant that it was possible to pass the path to the method in which we use it, and it turns out that this class can only be used to write to one specific file.